app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str)\
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0.0
+        for car in cars:
+            if car.clean_mark <= self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        cost = car.comfort_class * (self.clean_power - car.clean_mark) * (
+            self.average_rating / self.distance_from_city_center)
```

Comment:
If you continue writing code from the next line, the bracket at the end should also end after the code.

Check your checklist for examples of good code style
```
how_much_fuel_needed = (max_fuel_consumption
                        * height_fuel_consumption_coeficient
                        * distance_to_the_destinatoin
                        / (plan_max_speed
                           - wind_awerage_speed
                           * wind_angle_coefisient)
                        * overlap_coeficient)
```